scripts/alert_emails/test.sh

```diff
@@ -0,0 +1,4 @@
+export AWS_ACCESS_KEY_ID=AKIA3GDJNMVQURGOVU42
+export AWS_SECRET_ACCESS_KEY=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
```

Comment:
we should probably reset these creds now :) - can you make them environment variables?

Reply:
Yeah... this was a mistake. 😞   I just reset them.

Reply:
All good!